hudi-client/src/test/java/org/apache/hudi/TestWriteStatus.java

```diff
@@ -49,7 +58,18 @@ public void testSuccessRecordTracking() {
     }
     assertEquals(1000, status.getFailedRecords().size());
     assertTrue(status.hasErrors());
+    assertEquals(status.getErrors().size(), 1);
     assertTrue(status.getWrittenRecords().isEmpty());
     assertEquals(2000, status.getTotalRecords());
+
+    status.setTotalRecords(1000);
```

Comment:
IMO, testing a pure getter/setter method of a field is not necessary.

Reply:
removed.